conftest.py

```diff
@@ -200,8 +200,9 @@ def vm_refs(request):
 @pytest.fixture(scope="module")
 def imported_vm(host, vm_ref):
     if is_uuid(vm_ref):
-        print(">> Reuse VM %s on host %s" % (vm_ref, host))
         vm = VM(vm_ref, host)
+        name = vm.name()
```

Comment:
Why do you use a temporary variable in this context? :)

Reply:
Because the name() method does a ssh call so I wanted to avoid nesting it in another function call.

Reply:
Ok, it makes sense. :+1: